lib/plaid/api_client.rb

```diff
@@ -77,10 +77,13 @@ def call_api(http_method, path, opts = {})
             fail ApiError.new(:code => 0,
                               :message => response.return_message)
           else
+            raw_body = response.body
+            # Gracefully handle malformed body which is possible during API outage/maintanance
+            parsed_body = JSON.parse(raw_body) rescure nil
```

Comment:
Instead of the `rescue` shorthand, I think it would be better to rescue the `JSON::ParserError` class instead so it's explicit but will require that this be changed to use the multiline rescue form.

```suggestion
            parsed_body = JSON.parse(raw_body) rescue nil
```

Reply:
Makes sense, I'll change